magento/adobe-stock-integration
# #426 : JS error "accessing a crooss-origin frame" during authorization

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#426: JS error "accessing a crooss-origin frame" during authorization

### Manual testing scenarios (*)

1. Login to Admin panel
2. Open Adobe Stock grid (Catalog -> Categories -> Content -> Select from Gallery -> Search Adobe Stock)
3. Click "Sign in" link in top right corner to login to Adobe Stock account (or initiate image licensing process clicking "License" button in any image preview)
4. Expected Result: There are no js error in the console
